doc/src/manual/parallel-computing.md

```diff
@@ -284,15 +284,15 @@ snippet:
 
 ```julia-repl
 A = rand(10,10)
-remotecall_fetch(()->norm(A), 2)
+remotecall_fetch(()->sum(A), 2)
 ```
 
-In this case [`norm`](@ref) is a function that takes 2D array as a parameter, and MUST be defined in
-the remote process.  You could use any function other than `norm` as long as it is defined in the remote
+In this case [`sum`](@ref) is a function that takes 2D array as a parameter, and MUST be defined in
```

Comment:
Preexisting, but perhaps "that takes a {2D|two-dimensional} array as an argument"?

Reply:
It is unclear to me why that even matters, is it not enough to say "In this case the function `sum` MUST be defined in the remote process" ?

Reply:
Agreed; perhaps simplify these sentences accordingly then? :)